src/main.c

```diff
@@ -173,6 +217,59 @@ void WriteInterfaces()
     free(InterfaceArray);
 }
 
+inline int
+CombineMetadataWithPacket(
+    _In_ HANDLE File,
+    _In_ PCHAR FragBuf,
+    _In_ unsigned long FragLength,
+    _In_ long InterfaceId,
+    _In_ long IsSend,
+    _In_ long TimeStampHigh, // usec (unless if_tsresol is used)
+    _In_ long TimeStampLow,
+    _In_ PDOT11_EXTSTA_RECV_CONTEXT Metadata,
+    _In_ unsigned long ProcessId
+    )
+{
+
+    char Comment[MAX_PACKET_SIZE] = { 0 };
+    size_t CommentLength = 0;
+
+    int Err = NO_ERROR;
+
+    Err = StringCchPrintfA((char*)&Comment, MAX_PACKET_SIZE, "Packet Metadata: ReceiveFlags:0x%x, PhyType:%s, CenterCh:%u, NumMPDUsReceived:%u, RSSI:%d, DataRate:%u, PID=%d",
+        Metadata->uReceiveFlags,
+        DOT11_PHY_TYPE_NAMES[Metadata->uPhyId],
+        Metadata->uChCenterFrequency,
+        Metadata->usNumberOfMPDUsReceived,
+        Metadata->lRSSI,
+        Metadata->ucDataRate,
+        ProcessId);
+
+    if (Err != NO_ERROR) {
+        printf("Failed converting NdisMetadata to string with error: %u\n", Err);
+    }
+    else {
```

Comment:
} else {

Reply:
ok, yeah, this pattern wasn't what I had looked at earlier, but can do.

Reply:
btw, EVERY time I write a new line of code VS is changing these back to the way they were. So be aware that choosing this style is difficult to maintain.